cpp/src/arima/batched_kalman.cu

```diff
@@ -659,17 +666,67 @@ __global__ void confidence_intervals(const double* d_fc, const double* d_sigma2,
   d_upper[idx] = fc + margin;
 }
 
+void _lyapunov_wrapper(raft::handle_t& handle,
+                       const ARIMAMemory<double>& arima_mem,
+                       const MLCommon::LinAlg::Batched::Matrix<double>& A,
+                       MLCommon::LinAlg::Batched::Matrix<double>& Q,
+                       MLCommon::LinAlg::Batched::Matrix<double>& X, int r) {
+  if (r <= 5) {
```

Comment:
`MLCommon::LinAlg::Batched::b_lyapunov` also has a special branch for r<=5. Is the difference whether we do memory allocation here?